lib/console.js

```diff
@@ -35,26 +35,25 @@ function Console(stdout, stderr) {
   }
 }
 
-Console.prototype.log = function() {
-  this._stdout.write(util.format.apply(null, arguments) + '\n');
+Console.prototype.log = function(...args) {
+  this._stdout.write(`${util.format(...args)}\n`);
```

Comment:
I've separated the more general modifications to console out to a separate PR https://github.com/nodejs/node/pull/6233